source/versions/changes.rst

```diff
@@ -1,6 +1,33 @@
 Recent Changes
 ==============
 
+MindMeld 4.5
+-------------
+.. warning::
+
+   This release includes breaking changes. See below
+   for instructions on migrating your apps from MindMeld 4.4 to MindMeld 4.5
+
+**1. Support for Deep Learning Models **
+
+MindMeld now supports a [suite of deep neural architectures](https://www.mindmeld.com/docs/userguide/deep_neural_nets.html) for domain and intent classification as well as for
+entity recognition in its NLP pipeline. This also includes support for Pretrained Transformer Models (PTMs)
+available as part of the [Huggingface models hub](https://huggingface.co/models). Similar to their shallower counterparts,
+the deep models also support configurable parameters to alter the model architecture and the training process.
+
+
+**2. Active Learning for Entity Recognition **
+
+MindMeld's active learning tool (link: https://www.mindmeld.com/docs/userguide/active_learning.html) now has support for entity recognition models.
```

Comment:
This also should probably use the docs-internal link style

Reply:
Done!